wasmtest.sh

```diff
@@ -0,0 +1,237 @@
+#!/bin/bash
```

Comment:
That would be super helpful to add usage instruction as comments at the beginning of this file